conversion-measurement/publisher/views/_layout.pug

```diff
@@ -17,23 +17,25 @@ html
             h1 📰 Publisher site 
             div#menuWrapper
                 nav.menu
-                    div
-                        a(href='/') No ad
-                    div
-                        strong With an ad that can register clicks:
-                        div 
-                            a(href='/click-element') HTML element 
-                                code a
-                        div 
-                            a(href='/click-js') JS
-                    div
-                        strong With an ad that can register views:
-                        div 
-                            a(href='/view-element-img') HTML element 
-                                code img
-
-                        div 
-                            a(href='/view-js') JS
+                    div 
+                        strong You have the choice how the ad is included in the article below:
```

Comment:
```suggestion
                        strong Choose how the ad should be included in the article below:
```

For consistency with the phrasing you proposed on the ad side.
WDYT?

Reply:
Happy with those changes :)